src/components/HeroSlide.js

```diff
@@ -24,7 +29,15 @@ const HeroSlide = ({ src, alt, title, url, opacity, callBack }) => {
     }
   }
 
-  const image920 = imgSrc.replace('_1920.', '_920.')
+  useEffect(() => {
+    ;[mainImgRef, bkgdImgRef].forEach((imgRef) => {
+      if (imgRef.current && imgRef.current.complete) {
+        const originalSrc = imgRef.current.src
```

Comment:
I wonder if we could do this instead here:

`const { naturalWidth, naturalHeight } = imgRef.current
        setIsVertical(naturalWidth < naturalHeight)`

Reply:
This code has been moved into the useEffect now.